.github/workflows/ci.yml

```diff
@@ -189,27 +189,93 @@ jobs:
           echo $RESPONSE
           ACCOUNT_INDEX=$(echo $RESPONSE | jq '.[0]' || exit 1)
           echo "::set-output name=account_index::$(echo ${ACCOUNT_INDEX})"
-
-      - name: Run modules tests
-        timeout-minutes: 45
-        run: |
+          # TODO: Drop regions when multi-region is done
           regions=("ap-northeast-1" "ap-northeast-2" "ap-northeast-3" "ap-south-1" "ap-southeast-1" "ap-southeast-2" "ca-central-1" "eu-central-1" "eu-north-1" "eu-west-1" "eu-west-2" "eu-west-3" "sa-east-1" "us-east-2" "us-west-1" "us-west-2")
           regionslen=${#regions[@]}
           index=$(($RANDOM % $regionslen))
           region=${regions[$index]}
-          export AWS_REGION=$region
-          echo $AWS_REGION
+          echo "::set-output name=region::$(echo ${region})"
+
+      - name: Pre-clean Test Account
+        id: pre-clean-test-account
+        env:
+          IASQL_ENV: ci
+          ACCOUNT_INDEX: ${{ steps['determine-test-account'].outputs['account_index'] }}
+          AWS_REGION: ${{ steps['determine-test-account'].outputs['region'] }}
+          ACCESS_KEY_IDS: ${{ secrets.ACCESS_KEY_IDS }}
+          SECRET_ACCESS_KEYS: ${{ secrets.SECRET_ACCESS_KEYS }}
+        run: |
```

Comment:
it might make sense to break this up or put it in `scripts/` so we can reuse it in the other repo for prisma tutorial runs?

Reply:
I based this off of what we do in the prisma tutorial runs ;)